dashboard/test/ui/features/acquisition_products/pd/application_detail_view.feature

```diff
@@ -9,6 +9,7 @@ Feature: Teacher Application Detail View
 
     Then I click selector "table#summary-csp-teachers ~ .btn:contains(View all applications)"
     Then I wait until element "#status-filter" is visible
+    And I wait for 3 seconds
```

Comment:
Could we instead wait for the button to be visible? (similar to line 14)

Reply:
Yeah! I was going off of the flaky Facebook icon share test, but this feels more robust.

Reply:
Updated, and here's the [passing video](https://app.saucelabs.com/tests/05543f85bda04b32976c6bc7dddb9ae0#58).